.pre-commit-config.yaml

```diff
@@ -52,3 +52,16 @@ repos:
 #         - id: mypy
 #           args: [--no-strict-optional, --ignore-missing-imports]
 #           additional_dependencies: [types-PyYAML, types-requests]
+
+# Use to check python style
```

Comment:
please make the addition of black a separate PR, same with xmllint

Reply:
Done. I dumped them both into `add-black-formatter-for-ap-dds` and removed it from this PR.

Reply:
pre-commit changes I'd like to add at a later date:
```
# Use to check python style
  -   repo: https://github.com/psf/black
      rev: 23.1.0
      hooks:
        - id: black
          files: libraries/AP_XRCE_Client/wscript

  -   repo: https://github.com/lsst-ts/pre-commit-xmllint
      rev: v1.0.0
      hooks:
        - id: format-xmllint
          files: libraries/AP_XRCE_Client/dds_xrce_profile.xml
```